EventKit/eventkit-cloud
# Refactor dashboard to functional components

# Description of Improvement

Refactors the dashboard to use functional components, and updates tests to use react testing library.

Some bugs were found and fixed with testing components that use `withWidth` on export and react testing library. `withWidth` was removed from some components on export because of silent failure from react testing library (elements simply not included in DOM render, no error messages even with verbose output). `width` property had to be added manually where those components are used.

## How to test

Does the dashboard still render correctly?
Verify rendering when no data is present
Verify rendering when data is present
Verify notifications refresh as new packs are created + completed
Verify all tests pass

## Quality Assurance

The following items have been updated:


## Screenshots



:smile:
